mindsdb/api/mysql/mysql_proxy/data_types/mysql_packets/binary_resultset_row_package.py

```diff
@@ -83,6 +83,9 @@ def setup(self):
                 env_val = self.encode_time(val)
             elif col_type == TYPES.MYSQL_TYPE_NEWDECIMAL:
                 enc = 'string'
+            elif col_type == TYPES.MYSQL_TYPE_JSON:
```

Comment:
looks like this block doesn't do anything (enc is string anyway)
typo in `tthere`